opm/grid/cpgrid/Geometry.hpp

```diff
@@ -179,13 +181,27 @@ namespace Dune
             }
 
             /// This method is meaningless for singular geometries.
-            FieldMatrix<ctype, coorddimension, mydimension>
+            JacobianInverseTransposed
             jacobianInverseTransposed(const LocalCoordinate& /*local*/) const
             {
                 // Meaningless to call jacobianInverseTransposed() on singular geometries. But we need to make DUNE happy.
                 return FieldMatrix<ctype, coorddimension, mydimension>();
             }
 
+            /// This method is meaningless for singular geometries.
+            Jacobian
+            jacobian(const LocalCoordinate& /*local*/) const
+            {
+                return{};
```

Comment:
Should the previous two methods also be rewritten to just `return {};`? Nitpick: space after return.

Reply:
done.